src/blockchain/deposit_strategy/interface.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC, abstractmethod
+
+from blockchain.typings import Web3
+
+
+class ModuleDepositStrategyInterface(ABC):
```

Comment:
What is ABC here?

Reply:
ABC is mark for Abstract classes that should be implemented